### DISCUSSION TITLE
Issues with hostname config

### DISCUSSION THREAD

**Discussion by massimogallina** [kairos-io/kairos#884] [Q&A]
I am having issues using the `hostname` configuration option as per [documentation](https://kairos.io/docs/reference/configuration/#hostname).
If I use this option the file `/etc/hosts` is not created.
I am testing with image **kairos-debian-v1.5.1-k3sv1.26.1+k3s1**.

Is this a known issue? A workaround can be the manually generation of  `/etc/hosts` using the `file` configuration option. Are there other ways to solve the issue?

**Comment by mudler** [comment]
Hey @massimogallina :wave: 

Can you try setting just:

```yaml
#cloud-config

hostname: "your_hostname_here"
```

If the problem persist please file [a bug](https://github.com/kairos-io/kairos/issues/new?assignees=mudler&labels=bug&template=bug_report.md&title=), this is known to work in other flavors just fine.

**Reply by massimogallina** [reply]
I have tried with `hostname: "your_hostname"` too. It set the hostname, but does not create the `/etc/hosts` file.

I will open a bug as suggested.
